pkg/handler/project/project.go

```diff
@@ -226,3 +227,133 @@ func (h *handler) Create(c *gin.Context) {
 
 	c.JSON(http.StatusOK, view.CreateResponse(view.ToCreateProjectDataResponse(p), nil, done(nil), nil))
 }
+
+// AddMemberToProject godoc
+// @Summary Add member to an existing project
+// @Description Add member to an existing project
+// @Tags Project
+// @Accept json
+// @Produce json
+// @Param Authorization header string true "jwt token"
+// @Param id path string true "Project ID"
+// @Param Body body CreateMemberInput true "Body"
+// @Success 200 {object} view.CreateMemberDataResponse
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 404 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /projects/{id}/members [post]
+func (h *handler) AddMemberToProject(c *gin.Context) {
+	var body CreateMemberInput
+	if err := c.ShouldBind(&body); err != nil {
```

Comment:
The difference of ShouldBind and ShouldBindJson

Reply:
updated